tests/ttnn/integration_tests/resnet/test_ttnn_functional_resnet.py

```diff
@@ -22,102 +24,43 @@
 from models.experimental.functional_resnet.tt import ttnn_functional_resnet
 
 
+def update_ttnn_module_args(ttnn_module_args):
+    ttnn_module_args["use_1d_systolic_array"] = ttnn_module_args.in_channels < 256
+    ttnn_module_args["enable_auto_formatting"] = ttnn_module_args.kernel_size < (7, 7)
+    ttnn_module_args["conv_blocking_and_parallelization_config_override"] = {"act_block_h": 32}
+
+
 def custom_preprocessor(model, name, ttnn_module_args):
     parameters = {}
-    if isinstance(model, BasicBlock):
+    if isinstance(model, torchvision.models.resnet.BasicBlock):
         ttnn_module_args.conv1["activation"] = "relu"  # Fuse relu with conv1
 
         conv1_weight, conv1_bias = fold_batch_norm2d_into_conv2d(model.conv1, model.bn1)
```

Comment:
I'm realizing that we don't have much documentation around the model_preprocessing.py in general.    For example fold_batch_norm2d_into_conv2d.   Do we need to start adding doc here?  Do we want everyone to feel free to just add functions like this to model_preprocessing.py going forward?

Reply:
We need to document our use cases. And the process will be the same as always. People need to write their versions and if they are general enough, then we merge them into model_preprocessing.py